src/components/sticky-object.js

```diff
@@ -0,0 +1,123 @@
+/* global THREE, CANNON, AFRAME */
+AFRAME.registerComponent("sticky-object", {
+  dependencies: ["body", "super-networked-interactable"],
+
+  schema: {
+    autoLockOnLoad: { default: false },
+    autoLockOnRelease: { default: false },
+    autoLockSpeedLimit: { default: 0.25 }
+  },
+
+  init() {
+    this._onGrab = this._onGrab.bind(this);
+    this._onRelease = this._onRelease.bind(this);
+    this._onBodyLoaded = this._onBodyLoaded.bind(this);
+  },
+
+  play() {
+    this.el.addEventListener("grab-start", this._onGrab);
+    this.el.addEventListener("grab-end", this._onRelease);
+    this.el.addEventListener("body-loaded", this._onBodyLoaded);
+  },
+
+  pause() {
+    this.el.removeEventListener("grab-start", this._onGrab);
```

Comment:
should `remove` be called in this case?

Reply:
Hmm... how come? I thought something that's stuck would remain stuck through the pause.